proc_T1CPMG.py

```diff
@@ -50,12 +50,16 @@
         s.ift('t2')
         if flat_echo:
            s['t2',16]=0
+           # {{{ (FROM REVIEW) find echo center no longer exists!
            avg_center=find_echo_center(s,fl=fl)
+           # }}}
```

Comment:
probably want to replace this with hermitian function?

Reply:
in general, when I write the comment as "FROM REVIEW" feel free to delete it when you edit the code.